### PR TITLE
Update in-cluster agent configuration

### PR DESCRIPTION
**Pull request by tfadeyi**
This updates the in-cluster agent configuration to fetch venafi enhanced issuers resources present in the cluster.

Signed-off-by: Oluwole Fadeyi <oluwole.fadeyi@jetstack.io>

### CODE REVIEW

**Review by j-fuentes — APPROVED** [review]

**Review by dbarranco — APPROVED** [review]
I think it looks good to me, thanks much!

**Comment by maelvls** [comment]
Hey. Is there a release of `jsctl` that contains this fix? It looks like v0.1.16 doesn't contain the fix. How can I help?

**Comment by tfadeyi** [comment]
We can make a new release, if it's urgent, i think new work jsctl are going to take a little bit

**Comment by irbekrm** [comment]
We should at least merge #81 first though

File: internal/cluster/templates/agent.yaml [hunk 1]
```diff
@@ -424,6 +448,20 @@ data:
           group: networking.istio.io
           version: v1alpha3
           resource: virtualservices
+    - kind: "k8s-dynamic"
+      name: "k8s/venaficlusterissuer"
+      config:
+        resource-type:
+          group: jetstack.io
+          version: v1alpha1
+          resource: venaficlusterissuer
```
Comment (maelvls) [inline internal/cluster/templates/agent.yaml:457]: I think this should be `venaficlusterissuers`, right?

Comment (maelvls) [reply internal/cluster/templates/agent.yaml:457]: I proposed a fix in https://github.com/jetstack/jsctl/pull/82.

**Pull request merged**